index.js

```diff
@@ -4,11 +4,31 @@ const fetch = require('node-fetch');
 
 const key = '17c5c4c6e845e73f4296fa83b502b87c';
 
-const keyword = process.argv.splice(2).join(' ') || 'Big Yellow Door';
+const args = (process.argv.splice(2).join(' ') || 'Big Yellow Door@Delhi').split('@');
+
+if (args.length == 1) {
+  args.splice(1, 0, 'Delhi');
+} else if (args.length == 2) {
+  if (args[0] === '' || args[1] === '') {
+    console.log('Usage: zom <keyword>[@<location>]');
+    return;
+  }
+} else {
+  console.log('Usage: zom <keyword>[@<location>]');
+  return;
+}
+
+const { location, keyword } = { location: args[1], keyword: args[0] };
```

Comment:
```javascript

if (process.argv.length === 2) {
  console.log('Usage: zom <keyword>[@<location>]');
  return;
}

const [
  keyword = 'Big Yellow Door',
  location = 'New Delhi',
] = process.argv
    .splice(2)        // get rid of ['node', 'file.js']
    .join(' ')        // join them with space
    .split('@')       // split them at @
    .map(e => e.trim) // trim
;
```

Reply:
Will `splice(2)` work for `zom <keyword>[@<location>]` or `./index.js <keyword>[@<location>]` as well?

Reply:
I think it would. Gotta test it.

Reply:
`process.argv.length === 2` be a valid situation for cases like `zom beer` as default location override the search. Hence, we cannot terminate before we can identify arguments after parsing through the entire query.

Reply:
Ah! Wait, my bad :sweat_smile: